components/TextFields/examples/experimental/InputTextFieldManualLayoutExampleViewController.m

```diff
@@ -278,27 +294,35 @@ - (void)updateButtonThemes {
   }];
 }
 
-- (void)updateTextFieldStates {
-  [self.allTextFields
-      enumerateObjectsUsingBlock:^(MDCInputTextField *textField, NSUInteger idx, BOOL *stop) {
-        textField.isErrored = self.isErrored;
-        BOOL isEven = idx % 2 == 0;
-        if (textField.isErrored) {
-          if (isEven) {
-            textField.leadingUnderlineLabel.text = @"Suspendisse quam elit, mattis sit amet justo "
-                                                   @"vel, venenatis lobortis massa. Donec metus "
-                                                   @"dolor.";
-          } else {
-            textField.leadingUnderlineLabel.text = @"This is an error.";
-          }
+- (void)updateTextFieldThemes {
```

Comment:
In an example code, it's typically easier to clients to learn how to implement a certain scenario when all example code of a single scenario is container within the same piece of code. This code seems to take a different route.